cuda_core/tests/test_module.py

```diff
@@ -6,15 +6,22 @@
 # this software and related documentation outside the terms of the EULA
 # is strictly prohibited.
 
+try:
+    from cuda.bindings import driver
+except ImportError:
+    from cuda import cuda as driver
```

Comment:
FYI this can be done with
```suggestion
from cuda.core.experimental._utils.cuda_utils import driver
```
(We might want to group all cuda.core imports to the group below.)